--- a/OpenEXR/IlmImf/ImfDwaCompressor.cpp
+++ b/OpenEXR/IlmImf/ImfDwaCompressor.cpp
@@ -161,3 +161,4 @@
 #include <cstddef>
+#include <cstdint>
 
 
--- a/OpenEXR/IlmImf/ImfHuf.h
+++ b/OpenEXR/IlmImf/ImfHuf.h
@@ -40,3 +40,5 @@
 #include "ImfExport.h"
 #include "ImfNamespace.h"
+#include <cstdint>
+
 
--- a/OpenEXR/IlmImf/ImfMisc.h
+++ b/OpenEXR/IlmImf/ImfMisc.h
@@ -53,2 +53,3 @@
 #include <cstddef>
+#include <cstdint>
 #include <vector>


